ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2633,6 +2633,11 @@ router_bgp:
       local_as: < bgp_as >
       description: "< description as string >"
       shutdown: < true | false >
+      remove_private_as:
```

Comment:
Just a suggestion to consider. But feel free to decline:

```yaml
remove_private_as:
  egress: < enabled | disabled | all >
  ingress: < enabled | disabled | replace_as >
```

Apart from being more compact and readable, it prevents cases where all is defined without enabled true.
As an alternative, I'd suggest to treat `all: true` as `enabled: true` and `replace_as: true` as `ingress: true`

Reply:
Thank you for the suggestion. I like this model. Will implement this.

Reply:
The problem was with egress keyword. Because I think we need to implement something close to the cli match. But as I don't see any better option I will go with this.

Reply:
The EOS CLI mentions `outbound` for the regular mode and inbound for the `ingress` keyword.
Remember that `neighbor 1.2.3.4 remove-private-as all replace-as` is also valid CLI - not covered in Petr's example.

Reply:
Alternative could be:
```yaml
remove_private_as:
  enabled:
  all:
  replace_as:
remove_private_as_ingress:
  enabled:
  replace_as:
```

Reply:
OK, if we want to keep enbled true | false approach here is the suggestion:

```yaml
remove_private_as:
  in:
    enabled:
    replace_as:
  out:
    enabled:
    all:
    replace_as:
```

But I'm fine with the option suggested by Claus as well. The in/out, ingress/egress, inbound/outbound makes it more explicit. Which is good in my opinion. If CLI isn't explicit enough, we can still add some clarity in the data model if it's the right thing to do.

Reply:
I think it should state ingress in the name to reflect the config key being applied.  Claus model is my preference because it would be closer to the actual config being modeled.

Reply:
I don't think this will make it to 3.4.0. I am preparing release-notes etc now, so train has left the station :)

Reply:
Even I can't push it now.